src/app/(authenticated)/(dashboard)/[publicId]/documents/page.tsx

```diff
@@ -1,30 +1,16 @@
-import { db } from "@/server/db";
 import DocumentsTable from "./table";
 import DocumentUploadModal from "./modal";
 import { Card } from "@/components/ui/card";
 import { Button } from "@/components/ui/button";
-import { withServerSession } from "@/server/auth";
-import type { Document, User } from "@prisma/client";
+
 import EmptyState from "@/components/shared/empty-state";
 import { RiUploadCloudLine, RiAddFill } from "@remixicon/react";
-
-interface DocumentType extends Document {
-  uploadedBy: User | null;
-}
-
-const getDocuments = async (companyId: string) => {
-  return await db.document.findMany({
-    where: { companyId },
-    include: {
-      uploadedBy: true,
-    },
-  });
-};
+import { api } from "@/trpc/server";
```

Comment:
both does the job . They're colocated into tRPC because when tRPC 11 stabilizes, while calling a tRPC procedure (basically a fetch request)  RSC can deduplicate requests while fetching the same data (since next.js monkey patches fetch).